chunk/chunk_store.go

```diff
@@ -852,6 +853,16 @@ type dynamoBatchWriteItemsOp struct {
 	done      chan error
 }
 
+func nextBackoff(lastBackoff time.Duration) time.Duration {
+	// Based on the "Decorrelated Jitter" approach from https://www.awsarchitectureblog.com/2015/03/backoff.html
+	// sleep = min(cap, random_between(base, sleep * 3))
```

Comment:
So the way to comment Go code to is to write the Python equivalent in comments? :trollface: